packages/cli/src/NodeTypes.ts

```diff
@@ -8,12 +8,19 @@ import type {
 } from 'n8n-workflow';
 import { NodeHelpers } from 'n8n-workflow';
 import { Service } from 'typedi';
-import { RESPONSE_ERROR_MESSAGES } from './constants';
 import { LoadNodesAndCredentials } from './LoadNodesAndCredentials';
 import { join, dirname } from 'path';
 import { readdir } from 'fs/promises';
 import type { Dirent } from 'fs';
 
+class UnrecognizedNodeError extends Error {
+	severity = 'warning';
+
+	constructor(nodeType: string) {
+		super(`Node not recognized: ${nodeType}".`);
```

Comment:
this will create a distinct error group on sentry for every node type. is that desired?

Reply:
But we're skipping reporting on `warning` sev?

Reply:
ah. never mind. I've got cotton brain again.